stdlib/database/sql/src/main/ballerina/src/sql/error.bal

```diff
@@ -14,6 +14,23 @@
 // specific language governing permissions and limitations
 // under the License.
 
+public const BATCH_EXECUTE_ERROR_REASON = "{ballerina/sql}BatchExecuteError";
```

Comment:
Missing docs.

Reply:
These const are used for internal error creation and will be removed after the error syntax migration is complete